Add documentation for CloudWatch Logs service in AWS-S3 module (#2589)
wazuh/wazuh-documentation: source/user-manual/reference/ossec-conf/wodle-s3.rst

+--------------------+----------------------------------------+

service type
^^^^^^^^^^^^

Defines a service to process. Must have the attribute ``type`` defined. (Supports multiple instances of this option).

Service options
~~~~~~~~~~~~~~~

- `Service\\aws_account_id`_
- `Service\\aws_account_alias`_
- `Service\\aws_log_groups`_
- `Service\\access_key`_
- `Service\\secret_key`_
- `Service\\aws_profile`_
- `Service\\iam_role_arn`_
- `Service\\only_logs_after`_
- `Service\\regions`_
- `Service\\remove_log_streams`_

Service\\aws_account_id
^^^^^^^^^^^^^^^^^^^^^^^

The AWS Account ID for accesing the service.

+--------------------+-------------------------------------------+
| **Default value**  | All accounts.                             |
+--------------------+-------------------------------------------+

> | **Allowed values** | Comma list of 12 digit AWS Account ID     |

Author: changed to `Comma-delimited list`

Service\\secret_key
^^^^^^^^^^^^^^^^^^^

The secret key created for the IAM user with the permission to access the service.

+--------------------+--------------------------+
| **Default value**  | N/A                      |
+--------------------+--------------------------+
| **Allowed values** | Any alphanumerical key.  |
+--------------------+--------------------------+

Service\\aws_profile
^^^^^^^^^^^^^^^^^^^^

A valid profile name from a Shared Credential File or AWS Config File with the permission to access the service.

+--------------------+--------------------+
| **Default value**  | N/A                |
+--------------------+--------------------+
| **Allowed values** | Valid profile name |
+--------------------+--------------------+

Service\\iam_role_arn
^^^^^^^^^^^^^^^^^^^^^

A valid role arn with permission to access the service.

+--------------------+----------------+
| **Default value**  | N/A            |
+--------------------+----------------+
| **Allowed values** | Valid role arn |
+--------------------+----------------+

Service\\only_logs_after
^^^^^^^^^^^^^^^^^^^^^^^^

.. versionadded:: 4.0.0

> A valid date, in YYYY-MMM-DD format, that only logs from after that date will be parsed.  All logs from before that date will be skipped. Only works for CloudWatch Logs service.

Reviewer: ```suggestion
A valid date, in YYYY-MMM-DD format. Only those logs from after that date will be parsed, the logs from before that date will be skipped. Only works for CloudWatch Logs service.
```

> A valid date, in YYYY-MMM-DD format, that only logs from after that date will be parsed.  All logs from before that date will be skipped. Only works for CloudWatch Logs service.

Author: This format was chosen because is the one currently used for the rest of the supported buckets and services. We should not use a different format for each supported service. Changing all of them to use a new format will have negative impact on Wazuh users.

+--------------------+-------------+
| **Default value**  | 1970-JAN-01 |
+--------------------+-------------+
| **Allowed values** | Valid date  |
+--------------------+-------------+

Service\\regions
^^^^^^^^^^^^^^^^

.. versionadded:: 4.0.0

A comma-delimited list of regions to limit parsing of logs. Only works for CloudWatch Logs service.

+--------------------+----------------------------------------+
| **Default value**  | All regions                            |
+--------------------+----------------------------------------+
| **Allowed values** | Comma-delimited list of valid regions  |
+--------------------+----------------------------------------+

Service\\remove_log_streams
^^^^^^^^^^^^^^^^^^^^^^^^^^^

.. versionadded:: 4.0.0

> Define if you want to remove the log streams from the log groups after they are read by the wodle. Only works for CloudWatch Logs service.

Reviewer: ```suggestion
Define whether or not to remove the log streams from the log groups after they are read by the wodle. Only works for CloudWatch Logs service.
```

> Define whether or not to remove the log streams from the log groups after they are read by the wodle. Only works for CloudWatch Logs service.

Reviewer: ```suggestion
Define whether or not to remove the log streams from the log groups after they are read by the module. Only works for CloudWatch Logs service.
```
